invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/DocumentMetadataTabs.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
```

Comment:
There is another component also called `DocumentMetadataTabs` and looks very similar.

Reply:
yes, thanks, mistake with a rebase